mono/metadata/exception.c

```diff
@@ -1438,3 +1453,11 @@ mono_error_set_argument_out_of_range (MonoError *error, const char *name)
 	//FIXMEcoop
 	mono_error_set_exception_instance (error, mono_get_exception_argument_out_of_range (name));
 }
+
+MonoExceptionHandle
+mono_error_convert_to_exception_handle (MonoError *error)
+{
+	//FIXMEcoop mono_error_convert_to_exception is raw pointer
+	HANDLE_FUNCTION_ENTER ();
+	HANDLE_FUNCTION_RETURN_REF (MonoException, mono_error_convert_to_exception (error));
```

Comment:
`MONO_HANDLE_NEW` would have been enough.

Reply:
Agreed, having double checked the two existing calls.